torq.sh

```diff
@@ -241,15 +240,15 @@ elif [[ "$1" == "stop" ]]; then
   stopprocs $PROCS;
 elif [[ "$1" == "summary" ]]; then
   allcsv $@;
-  PROCS=$(getall);
+  PROCS=`awk -F, '{if(NR>1) print $4}' $CSVPATH`;
```

Comment:
Seems like this global is set a bunch of different places under different conditions, but all in the same way - could this just be set once when the script starts & used wherever it's needed?